src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -31,20 +33,22 @@ public class AddCommandParser implements Parser<AddCommand> {
      */
     public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_TAG);
+                ArgumentTokenizer.tokenize(args, PREFIX_UID, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL,
```

Comment:
wait since there is a prefix means like the users are going to type in the uid themselves? for addcommand

Reply:
hmm maybe instead like this, can be like during the parse we will retrive the uid internally from the uidmanager class then like use that uid to create Person(uid, .... sth like this)